pkg/cli/cli.go

```diff
@@ -214,11 +216,25 @@ func (c *CLI) getInfoFromConfigFile() error {
 	return c.getInfoFromConfig(cfg.Config())
 }
 
+// getInfoFromConfigFilePath obtains the project version and plugin keys from the project config file via a path.
+func (c *CLI) getInfoFromConfigFilePath(path string) error {
+	if path == "" {
+		return c.getInfoFromConfigFile()
+	}
+	cfg := yamlstore.New(c.fs)
+	if err := cfg.LoadFrom(path); err != nil {
+		return err
+	}
+
+	return c.getInfoFromConfig(cfg.Config())
+}
```

Comment:
The project has interfaces for we are able to load and parse the PROJECT file to its structure already, see: 

https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/config/v3/config.go
https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/config/store/interface.go